catalog-rest-service/src/main/resources/json/schema/entity/data/table.json

```diff
@@ -274,57 +274,65 @@
           "description": "Column Name.",
           "type": "string"
         },
-        "valuesCount": {
+        "count": {
           "description": "Total count of the values in this column.",
-          "type": "number"
```

Comment:
reason to keep number is the python generated code will make it int and the range for the int in python is lower compare to java